tests/test_app.rs

```diff
@@ -2571,17 +2571,192 @@ fn test_security_app() {
     );
 }
 
+#[test]
+fn test_method_macro() {
+    #[get("/v0/pets")]
+    #[api_v2_operation]
```

Comment:
That's neat! The only thing I'm wondering is if `api_v2_operation` macro is not defined here I guess we should pretend it is...

Reply:
Yeah the `route` (get/put...) macros could automatically add the `#[api_v2_operation]`, not exactly sure how to do that yet but it's probably possible!
Oh and what about adding the `route` to the api macro, as such:
`#[api_v2_operation(get("/v0/pets"))]` ?
Though the downside is more "labour" converting things. Any thoughts?

Reply:
Better to keep compatibility with established `actix_web_macros` definitions, main idea is to make a transition to use paperclip as easy as possible.

Reply:
Yeah agreed - I'll see if I can detect the existence of the `api_v2_operation` :+1:

Reply:
Sadly I do not know how to do this :( 
I tried using syn::parse_outer which parses a macro just fine, but because the item also has the function it fails...

Reply:
I think that's fine , by itself this change is valuable as enables paperclip for usage with actix-web-macros